src/components/common/icons/token_icon.tsx

```diff
@@ -69,9 +70,31 @@ export const TokenIcon = (props: Props) => {
     const { token } = props;
     const TokenIconComponentName = getTokenIconNameBySymbol(token.symbol) as keyof typeof TokenIcons;
     const Icon: React.FunctionComponent = TokenIcons[TokenIconComponentName];
-    return Icon ? React.createElement(Icon, null, null) : null;
+    return Icon
+        ? React.createElement(Icon, null, null)
+        : fallbackIcon({
+              fill: (token && token.primaryColor) || DEFAULT_FALLBACK_ICON_COLOR,
+              title: token.symbol && token.symbol.toUpperCase(),
+          });
 };
 
 const getTokenIconNameBySymbol = (symbol: string): string => {
     return `${symbol.charAt(0).toUpperCase()}${symbol.slice(1)}TokenIcon`;
 };
+
+const fallbackIcon = ({
```

Comment:
Can we convert this into a `FallbackIcon` component? I think we just have to change its name and how it's used above.

Reply:
Done